vigor/views.py

```diff
@@ -4,30 +4,45 @@
 from .forms import UserCreationFormExtended
 from django.contrib.auth import authenticate
 from django.contrib.auth import login as auth_login
-
-
 from django.views.decorators.csrf import csrf_exempt
+from .models import User
 
 def index(request):
     template = loader.get_template("vigor/index.html")
     context = {}
     return HttpResponse(template.render(context, request))
 
 def login(request):
-    return HttpResponse("Login Page.")
+    if request.user.is_authenticated():
+        return HttpResponseRedirect("/vigor/dashboard")
+
+    if request.method == 'POST':
+        username = request.POST['username']
+        password = request.POST['password']
+        user = authenticate(username=username, password=password)
+        if user is not None:
+            auth_login(request, user)
+            return HttpResponseRedirect("/vigor/dashboard") # change to just `/dashboard` when in production
+        else:
+            print("Internal Server Error")
```

Comment:
yeah, this will come in due time. I'm just not sure of all the checks I have to do, other than the one that is implied in that the user doesn't actually exist yet. So, maybe we can put an error message that says that the user with those credentials cannot be found in the db?